capa/v25.0.0/README.md

```diff
@@ -0,0 +1,162 @@
+# :zap: Giant Swarm Release v25.0.0 for CAPA :zap:
+
+We are happy to announce our first `Cluster API for AWS` (CAPA) release v25. 
+
+This is the first Giant Swarm supported CAPA release. It is available on CAPA Management Clusters and will be used as a first release to be upgraded to from `Vintage` workload clusters.
+The Vintage clusters developments end with major release `v20.x`. Going forward all the major CAPA releases will be in sync with minor Kubernetes version, kicking off the new versioning of the releases with Kubernetes `v1.25.16` and `v25.0.0` for CAPA.
+
+## CAPA benefits
+
+Each existing customer using the Giant Swarm Vintage AWS product has been given a presentation about CAPA benefits. We have gathered the most crucial high-level advantages in the list below to refresh your memory:
```

Comment:
> to refresh your memory
I wouldn't include in the release notes.

Reply:
fixed